### PR TITLE
max_line_length: changed to 79

### PR DESCRIPTION
**Pull request by Ana06**
The attribute `max_line_length` changed to 79 and all the lines that have 80 characters fixed to follow that.

Closes https://github.com/coala-analyzer/coala-bears/issues/580

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
end commit message body with period please

File: bears/general/IndentationBear.py [hunk 1]
```diff
@@ -222,7 +223,8 @@ def get_indent_levels(self,
                 if _range.start.line == line + 1:
                     next_indent += 1
 
-                first_ch = file[line].lstrip()[0] if file[line].lstrip() else ""
+                first_ch = file[line].lstrip()[0] if file[
+                                             line].lstrip() else ""
```
Comment (Adrianzatreanu) [inline bears/general/IndentationBear.py]: id break it before the else, is that doable?

File: bears/general/InvalidLinkBear.py [hunk 2]
```diff
@@ -63,9 +63,9 @@ def run(self, filename, file,
         This bear can automatically fix redirects, but ignores redirect
         URLs that have a huge difference with the original URL.
 
-        Warning: This bear will make HEAD requests to all URLs mentioned in your
-        codebase, which can potentially be destructive. As an example, this bear
-        would naively just visit the URL from a line that goes like
+        Warning: This bear will make HEAD requests to all URLs mentioned in
+        yourcodebase, which can potentially be destructive. As an example, this
```
Comment (Adrianzatreanu) [inline bears/general/InvalidLinkBear.py]: space between your and codebase?

**Comment by sils1297** [comment]
also commit message isn't present tense

File: bears/go/GoErrCheckBear.py [hunk 3]
```diff
@@ -40,7 +41,8 @@ def create_arguments(filename, file, config_file,
                              regex apply to all packages.
         :param ignorepkg:    Takes a comma-separated list of package
                              import paths to ignore.
-        :param asserts:      Enables checking for ignored type assertion results
+        :param asserts:      Enables checking for ignored type assertion
+                             results
```
Comment (Adrianzatreanu) [inline bears/go/GoErrCheckBear.py]: could you add a period here in a separate commit please :D ? now i notice it, thanks

**Comment by Ana06** [comment]
@Adrianzatreanu 

> id break it before the else, is that doable?

I think I have to add `\` to break before the else, is that okey?

**Comment by sils1297** [comment]
@Ana06 we usually avoid breaking with `\` one can wrap a string or a condition in parentheses and then you can avoid breaking with backslash. This is not a hard requirement ATM though I think, @Makman2 wanted it but never wrote it down :P

**Comment by sils1297** [comment]
can you start the first letter after the colon with upper case? `something: Something`, we use this consistently

**Comment by sils1297** [comment]
unack https://github.com/coala-analyzer/coala-bears/pull/657/commits/ff63fdf18e368c17e8fcc6d7272fd610eb1b6dfc , except commit message it's good though

**Comment by Makman2** [comment]
\+ for the period commit you should choose as a tag `GoErrCheckBear`, as this is the file you change :)

**Comment by Ana06** [comment]
@sils1297 
> @Ana06 we usually avoid breaking with \ one can wrap a string or a condition in parentheses and then you can avoid breaking with backslash.

I am not sure how to avoid using `\` here if you want to break before the else: 

``` Python
first_ch = file[line].lstrip()[0] if file[line].lstrip() \
           else ""
```

:confused:

**Comment by hypothesist** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/657/commits/5f50e585e4dd0895c9850b4514312fd2c9c102c8

**Comment by sils1297** [comment]
I'd do

```
first_ch = (file[line].lstrip()[0]
           if file[line].lstrip() else "")
```

instead

File: bears/general/IndentationBear.py [hunk 4]
```diff
@@ -222,7 +223,8 @@ def get_indent_levels(self,
                 if _range.start.line == line + 1:
                     next_indent += 1
 
-                first_ch = file[line].lstrip()[0] if file[line].lstrip() else ""
+                first_ch = file[line].lstrip()[0] if file[line].lstrip() \
+                    else ""
```
Comment (tushar-rishav) [inline bears/general/IndentationBear.py]: @Ana06 @Adrianzatreanu This is one way. 
```python
first_ch = (file[line].lstrip()[0]
            if file[line].lstrip()
            else "")
```
Note: For demo it has been shown in 3 lines. You may go with two lines only, keeping else in second line. Personally, I like 3 lines split style. :)

**Comment by sils1297** [comment]
or rather do `file[line].lstrip()[:1]` then you don't need an if whatsoever :)

File: bears/general/InvalidLinkBear.py [hunk 5]
```diff
@@ -105,7 +106,8 @@ def run(self, filename, file,
                         end = start + len(link)
                         replacement = current_line[:start] + \
                             redirect_url + current_line[end:]
-                        diff.change_line(line_number, current_line, replacement)
+                        diff.change_line(
+                            line_number, current_line, replacement)
```
Comment (tushar-rishav) [inline bears/general/InvalidLinkBear.py]: @Ana06 
I believe this won't exceed the limit.
```python
diff.change_line(line_number,
                 current_line,
                 replacement)
```

File: bears/general/InvalidLinkBear.py [hunk 6]
```diff
@@ -94,7 +94,8 @@ def run(self, filename, file,
                         line=line_number,
                         severity=RESULT_SEVERITY.NORMAL)
                 if 300 <= code < 400:  # HTTP status 30x
-                    redirect_url = requests.head(link, allow_redirects=True).url
+                    redirect_url = requests.head(
+                        link, allow_redirects=True).url
```
Comment (tushar-rishav) [inline bears/general/InvalidLinkBear.py]: I am not sure aligning column wise would exceed the limit here? If it doesn't then maybe we should prefer doing it.

File: bears/vcs/git/GitCommitBear.py [hunk 7]
```diff
@@ -121,7 +121,8 @@ def check_shortlog(self, shortlog,
 
         if shortlog_imperative_check:
             colon_pos = shortlog.find(':')
-            shortlog = shortlog[colon_pos + 1:] if colon_pos != -1 else shortlog
+            shortlog = shortlog[colon_pos +
+                                1:] if colon_pos != -1 else shortlog
```
Comment (tushar-rishav) [inline bears/vcs/git/GitCommitBear.py]: Maybe we can make it better by:
```python
shortlog = (shortlog[colon_pos + 1:]
            if colon_pos != -1
            else shortlog)
```

File: tests/python/RadonBearTest.py [hunk 8]
```diff
@@ -14,7 +14,8 @@ class class1():
 
 
 RadonBearNoReportsTest = verify_local_bear(RadonBear,
-                                           valid_files=(test_file1, test_file2),
+                                           valid_files=(
+                                               test_file1, test_file2),
```
Comment (tushar-rishav) [inline tests/python/RadonBearTest.py]: Align column wise. So in general if inline param style exceeds the limit then it'd be nice to align the params column wise. Similar changes at other places too :+1:

**Comment by tushar-rishav** [comment]
unack 0db9f9ac1433c4395346c5ab6260450a447d7e29

Need some minor changes! :)

**Comment by tushar-rishav** [comment]
unack 0db9f9ac1433c4395346c5ab6260450a447d7e29

**Comment by tushar-rishav** [comment]
ack 2b3e367d6b950265f4cd797e27a7e037e64c5bfc d2e513a602442a022aeb6de2541d330d37ed3f97 f8e1179fa8a8c3d47f93299ce02316976b63d415

**Comment by tushar-rishav** [comment]
@rultor merge

**Comment by tushar-rishav** [comment]
@rultor merge

**Comment by hypothesist** [comment]
@rultor merge

**Comment by sils1297** [comment]
Read the message: `fatal: Not possible to fast-forward, aborting.`

**Comment by sils1297** [comment]
so just a simple rebase will sole this

**Comment by hypothesist** [comment]
@sils1297 Ah I wonder Github doesn't say that in the status panel though

**Comment by tushar-rishav** [comment]
reack https://github.com/coala-analyzer/coala-bears/pull/657/commits/921322ece71ce797d67b82bb272392f03e0868cb https://github.com/coala-analyzer/coala-bears/pull/657/commits/eac209443bc5dd3a93cf2c6c42d3f712189d601e https://github.com/coala-analyzer/coala-bears/pull/657/commits/246ca50f91a8c2d23492043259961affc3a05f50

**Comment by tushar-rishav** [comment]
@rultor merge

**Pull request merged**